storage/redis/store.go

```diff
@@ -0,0 +1,126 @@
+/*
+ * Flow Emulator
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package redis
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	redis "github.com/go-redis/redis/v8"
+	"github.com/onflow/flow-emulator/storage"
+)
+
+// Store implements the Store interface
+type Store struct {
+	storage.StoreImpl
+	options *redis.Options
+	rdb     *redis.Client
+}
+
+// New returns a new in-memory Store implementation.
+func New(url string) (*Store, error) {
+	options, err := redis.ParseURL(url)
+	if err != nil {
+		return nil, err
+	}
+	store := &Store{
+		options: options,
+		rdb:     redis.NewClient(options),
+	}
+	store.DataBackend = store
+	store.KeysBackend = &storage.StorageBackendKeysImpl{}
+
+	return store, nil
+}
+
+func (s *Store) GetBackend() storage.StorageBackendData {
```

Comment:
I'm not sure if this is supposed to be exported.

Reply:
I think it can even be deleted, it is left over somehow I missed.